LondonBritishLibrary/orient/BLorient765.xml

```diff
@@ -30,20 +30,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 261</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
-                    <msContents>
-                        <summary/>
-                        
```

Comment:
as always, I am in favour of the empty element, as it lets the overall summary be generated and printed